doc/source/whatsnew/v0.16.1.txt

```diff
@@ -135,6 +135,47 @@ values NOT in the categories, similarly to how you can reindex ANY pandas index.
 
 See the :ref:`documentation <advanced.categoricalindex>` for more. (:issue:`7629`)
 
+.. _whatsnew_0161.enhancements.sample:
```

Comment:
Can you add highlights description, linking to this section, like:

- https://github.com/pydata/pandas/blob/master/doc/source/whatsnew/v0.16.1.txt#L12